config/config.go

```diff
@@ -73,6 +72,18 @@ func (ss *strs) Set(str string) error {
 	return nil
 }
 
+// Hudson consensus config
+const (
```

Comment:
As we already tested 5 sec is reliable. what if we directly update the original configs? These are node configs, and doesn't need consensus on them.